Tests/UnitTests/Identity/IdentityManagerTests.swift

```diff
@@ -116,10 +116,8 @@ class IdentityManagerTests: TestCase {
         expect(self.mockDeviceCache.invokedClearCachesForAppUserID) == false
         expect(self.mockBackend.invokedClearHTTPClientCaches) == true
         expect(self.mockBackend.invokedClearHTTPClientCachesCount) == 1
-        expect(self.mockDeviceCache.invokedClearCustomerInfoCache) == true
```

Comment:
@tonidero, instead of removing this line, shouldn't we check here this? 
```
expect(self.mockDeviceCache.invokedClearCustomerInfoCache) == false
```
To make sure that the customer info cache was not cleared

Reply:
Good point! Yeah, that's better will do the change, thanks!

Reply:
Done in https://github.com/RevenueCat/purchases-ios/pull/4671/commits/496b035274038cd4eeaa524abe1f783b62dc64ee